system/libs/pot/OTS_Player.php

```diff
@@ -2231,6 +2231,16 @@ public function setBlessings($blessings)
         $this->data['blessings'] = (int) $blessings;
     }
 
+	public function checkBlessings()
+    {
+        if( !isset($this->data['id']) )
+        {
+            throw new E_OTS_NotLoaded();
+        }
+		$value = $this->db->query('SELECT `blessings1`,`blessings2`,`blessings3`,`blessings4`,`blessings5`,`blessings6`,`blessings7`,`blessings8` FROM ' . $this->db->tableName('players') . ' WHERE ' . $this->db->fieldName('id') . ' = ' . $this->data['id'])->fetch();
```

Comment:
That will depends, on 10.x there is only 5 blessings and on the newest versions there is 8 blessings.

Also, it will return an array containing the blessings, I think it will be more efficient only returns if is true or false for the blessing - or the player has bless or don't.

Reply:
Hey vankk,
I didn't think about that.   I will look into another way of doing it., originally I had it check each blessing individually but then decided to do a single query.

Appreciate the feedback, I don't have a 5 blessing server and haven't played rl tibia in years, so wasn't sure when the change actually happened.